lib/ui/screens/wrapper.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+import 'package:zachranobed/ui/screens/login.dart';
+
+class Wrapper extends StatelessWidget {
+  const Wrapper({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // return either Home or Login widget
+    return Login();

```

Comment:
Tady je potreba implementovat Navigator (je to zakladni pattern ve Flutteru). Zkus sem klidne hodit ten Fake Login + Home screen.